Dockerfile

```diff
@@ -1,12 +1,14 @@
-FROM golang:1.20.2-alpine AS builder
+FROM golang:1.21.0-alpine3.18 as builder
```

Comment:
We are using go 1.20 currently.
In general you can leave the current dockerfile as is

Reply:
Removed changes from Dockerfile, but changed `kor` binary path so it can be on $PATH.

Reply:
Sure